Alpaca-Finance-Agents/HighShareLP/README.md

```diff
@@ -0,0 +1,22 @@
+# High Share Liquidity Pool Agent
+
+## Description
+
+This agent detects when a worker owns more than 51% of the shares in a pool. It is worth nothing that the agent does not scan every block because the computation does more requests than usual. In addition to this, the percentages computed will change very infrequently.
```

Comment:
```suggestion
This agent detects when a worker owns more than 51% of the shares in a pool. 
```
I would suggest to only keep this sentence. We usually write in the description only what the agent do, so the "how" or any other details can be removed